### ISSUE TITLE
In v1.1.0 last two skills in sort list are out of order

### ISSUE THREAD

**Issue by wduda** [wduda/TravelWindowII#137]
Minor Bug: "Guide to Estolad Lân" skill stays at the bottom of my self-sorted order no matter how I change it.

https://www.lotrointerface.com/downloads/fileinfo.php?id=1113#cmnt23653

**Comment by wduda** [comment]
> One thing I've noticed is whenever I set my milestone skill to either of the last two options the Travel Window II plugin doesn't correlate with the in game milestone order.

Basically what I mean is, lets say I set my milestone to the last on the list the Travel Window II plugin shows this in 2nd to last on the travel window.

**Comment by whiterabbit963** [comment]
I observed that v1.2.0 (and the fix_dictionary cleanup) appears to fix this issue. I assume that the duplicate names was causing selection issues.
To reproduce I unloaded TravelWindow, deleted the plugindata, loaded v1.1.0, opened the sort tab and moved Estolad Lan to the bottom. Instead Limlok was moved to the bottom.
The same process but with v1.2.0 works correctly.
I was unable to reproduce this issue as stated, but I can only assume this is due to having a different set of skills available.
